core/src/main/scala/org/apache/spark/scheduler/MapStatus.scala

```diff
@@ -141,7 +141,7 @@ private[spark] class HighlyCompressedMapStatus private (
     private[this] var numNonEmptyBlocks: Int,
     private[this] var emptyBlocks: RoaringBitmap,
     private[this] var avgSize: Long,
-    @transient private var hugeBlockSizes: Map[Int, Byte])
+    private[this] var hugeBlockSizes: Map[Int, Byte])
```

Comment:
if you can figure out a way to make it serializable with kryo and still keep the customized serialization logic for java serializer, I'm ok with it.